d2renderers/d2svg/d2svg.go

```diff
@@ -136,156 +137,175 @@ func arrowheadMarker(isTarget bool, id string, connection d2target.Connection) s
 	var path string
 	switch arrowhead {
 	case d2target.ArrowArrowhead:
-		attrs := fmt.Sprintf(`class="connection" fill="%s" stroke-width="%d"`, connection.Stroke, connection.StrokeWidth)
+		polygonEl := svgstyle.NewThemableElement("polygon")
+		polygonEl.Fill = svgstyle.ConnectionTheme(connection)
```

Comment:
this is being too presumptuous about `ConnectionTheme`. either the method shouldn't exist, or it has potential to be more than just the stroke color one day, in which case this would break. The fill should still just be `connection.Stroke`